ops/services/app_gateway/main.tf

```diff
@@ -28,6 +28,11 @@ resource "azurerm_application_gateway" "load_balancer" {
     subnet_id = var.subnet_id
   }
 
+  identity {
+    type = "UserAssigned"
+    identity_ids = [azurerm_user_assigned_identity.gateway.id]
```

Comment:
what does this do?

Reply:
This allows the gateway to access the secret values at runtime using its own permissions, rather than at creation time using the service account permissions. Similar to IAM roles in AWS, I think.